### DISCUSSION TITLE
安装了idm，但是百度网盘的链接打开一直显示未发现idm

### DISCUSSION THREAD

**Discussion by TyroneGao321** [syhyz1990/baiduyun#207] [Q&A]
![QQ截图20230311181640](https://user-images.githubusercontent.com/127592689/224476040-af2e2bd1-5419-475b-b44e-a788bb7e734a.png)

使用的是微软edge浏览器，插件安装了idm，但是基本所有链接都无法直接转到idm进行下载
这种问题该如何解决？

**Comment by 6Tom** [comment]
试一下用鼠标中键单击链接，而不是左键
